File: backend/app/plugins/simulators/tv/manifest.py
from app.plugins.base import PluginCategory, PluginManifest

manifest = PluginManifest(
    code="tv",
    name="TV / Medias",
    name_en="TV / Media",
    description="Simulation de chaines TV et segments media",
    category=PluginCategory.SIMULATOR,
    icon="Tv",
    default_color="purple",
    supported_formats=["video"],
    default_enabled=False,
    coming_soon=False,
    sort_order=3,
    inject_types=["tv"],
    router_module="app.routers.tv",
    router_prefix="/api/tv",
)

File: frontend/src/plugins/simulators/tv/manifest.ts
import { lazy } from 'react'
import { Tv } from 'lucide-react'
import type { PluginManifest } from '../../types'

export const manifest: PluginManifest = {
  code: 'tv',
  name: 'plugins.tv.name',
  description: 'plugins.tv.description',
  category: 'simulator',
  icon: Tv,
  defaultColor: 'purple',
  supportedFormats: ['video'],
  defaultEnabled: false,
  comingSoon: false,
  sortOrder: 3,
  injectTypes: ['tv'],
  playerRoute: '/tv',
  PlayerPage: lazy(() => import('./PlayerPage')),
  EventReceiver: lazy(() => import('./EventReceiver')),
  PreviewPopup: lazy(() => import('./PreviewPopup')),
  configSchema: null,
}
